### PR TITLE
Added DPTs based on KNX Specifications v2.1

### PR DESCRIPTION
**Pull request by eXtenZy**
For issue #236
Added all DPTs in the 5.*, 7.*, 8.*, 9.*, 13.* and 14.* categories based on the KNX Standard v2.1

### CODE REVIEW

File: xknx/dpt/dpt_4byte_int.py [hunk 1]
```diff
@@ -68,3 +68,58 @@ class DPT4ByteSigned(DPT4ByteUnsigned):
     resolution = 1
 
     _struct_format = ">i"
+
+
+class DPTValue4Count(DPT4ByteSigned):
+    """DPT 13,001 DPT_Value_4_Count (pulse)."""
```
Comment (farmio) [inline xknx/dpt/dpt_4byte_int.py]: Please replace `DPT 13,*` with `DPT 13.*`. (these are parsed in https://github.com/XKNX/xknx/blob/master/home-assistant-plugin/home-assistant.io/sensor_table_generator.py )

File: xknx/dpt/dpt_4byte_float.py [hunk 2]
```diff
@@ -45,43 +45,331 @@ def to_knx(cls, value):
             raise ConversionError("Cant serialize %s" % cls.__name__, vlaue=value)
 
 
+class DPTAcceleration(DPT4ByteFloat):
+    """DPT 14,000 DPT_Value_Acceleration (ms-2)."""
```
Comment (farmio) [inline xknx/dpt/dpt_4byte_float.py]: Please replace `DPT 14,*` with `DPT 14.*`. (these are parsed in https://github.com/XKNX/xknx/blob/master/home-assistant-plugin/home-assistant.io/sensor_table_generator.py )

File: xknx/dpt/dpt_4byte_float.py [hunk 3]
```diff
@@ -45,43 +45,331 @@ def to_knx(cls, value):
...
+
+
+class DPTAngularMomentum(DPT4ByteFloat):
+    """DPT 14,008 DPT_Value_Angular_Momentum (J s)."""
+
+    unit = "J s"
+
+
+class DPTAngularVelocity(DPT4ByteFloat):
+    """DPT 14,009 DPT_Value_Angular_Velocity """
```
Comment (farmio) [inline xknx/dpt/dpt_4byte_float.py]: Docstrings should end with a period. https://travis-ci.org/XKNX/xknx/jobs/637620014?utm_medium=notification&utm_source=github_status

File: xknx/dpt/__init__.py [hunk 4]
```diff
@@ -9,20 +9,36 @@
 from .dpt_1byte_signed import (
     DPTPercentV8, DPTSignedRelativeValue, DPTValue1Count)
 from .dpt_1byte_uint import (
-    DPTPercentU8, DPTSceneNumber, DPTTariff, DPTValue1Ucount)
+    DPTPercentU8, DPTDecimalFactor, DPTSceneNumber, DPTTariff, DPTValue1Ucount)
 from .dpt_2byte_float import (
-    DPT2ByteFloat, DPTEnthalpy, DPTHumidity, DPTLux, DPTPartsPerMillion,
-    DPTPressure2Byte, DPTTemperature, DPTVoltage, DPTWsp)
+    DPT2ByteFloat, DPTCurrent, DPTEnthalpy, DPTHumidity, DPTKelvinPerPercent, DPTLux, DPTPartsPerMillion, DPTPower,
```
[the remark this answers is not in the document]
Comment (eXtenZy) [reply xknx/dpt/__init__.py]: Added a "2Byte" suffix in a later commit so as to not break the setup of users already using this DPT.

File: xknx/dpt/__init__.py [hunk 5]
```diff
@@ -9,20 +9,36 @@
...
+    DPTPowerDensity, DPTPressure2Byte, DPTRainAmont, DPTTemperature, DPTTemperatureA, DPTTemperatureDifference,
```
[the remark this answers is not in the document]
Comment (eXtenZy) [reply xknx/dpt/__init__.py]: Same here: Added a "2Byte" suffix in a later commit so as to not break the setup of users already using this DPT.

File: xknx/dpt/__init__.py [hunk 6]
```diff
@@ -9,20 +9,36 @@
...
+    DPTElectricCharge, DPTElectricCurrent, DPTElectricCurrentDensity, DPTElectricDipoleMoment, DPTElectricDisplacement,
+    DPTElectricFieldStrength, DPTElectricFlux, DPTElectricFluxDensity, DPTElectricPolarization, DPTElectricPotential,
+    DPTElectricPotentialDifference, DPTElectromagneticMoment, DPTElectromotiveForce, DPTEnergy, DPTForce, DPTFrequency,
+    DPTAngularFrequency, DPTHeatCapacity, DPTHeatFlowRate, DPTHeatQuantity, DPTImpedance, DPTLength, DPTLightQuantity,
+    DPTLuminance, DPTLuminousFlux, DPTLuminousIntensity, DPTMagneticFieldStrength, DPTMagneticFlux,
+    DPTMagneticFluxDensity, DPTMagneticMoment, DPTMagneticPolarization, DPTMagnetization, DPTMagnetomotiveForce,
+    DPTMass, DPTMassFlux, DPTMomentum, DPTPhaseAngleRad, DPTPhaseAngleDeg, DPTPower, DPTPowerFactor, DPTPressure,
+    DPTReactance, DPTResistance, DPTResistivity, DPTSelfInductance, DPTSolidAngle, DPTSoundIntensity, DPTSpeed,
+    DPTStress, DPTSurfaceTension, DPTCommonTemperature, DPTAbsoluteTemperature, DPTTemperatureDifference,
+    DPTThermalCapacity, DPTThermalConductivity, DPTThermoelectricPower, DPTTime, DPTTorque, DPTVolume, DPTVolumeFlux,
```
[the remark this answers is not in the document]
Comment (eXtenZy) [reply xknx/dpt/__init__.py]: Yeah, I don't have a good idea of what to name this. I renamed it to DPTTimeSeconds, but I am open to suggestions.

File: xknx/dpt/dpt_4byte_int.py (hunk 1 above)
Comment (eXtenZy) [reply xknx/dpt/dpt_4byte_int.py]: Will do.

File: xknx/dpt/dpt_4byte_float.py (hunk 2 above)
Comment (eXtenZy) [reply xknx/dpt/dpt_4byte_float.py]: Will do.

**Comment by eXtenZy** [comment]
I've fixed some the points in a later commit.
I didn't know about the remote_value_sensor, I'll add the dpt classes in there as well, but I have to look at the code a bit.

**Comment by farmio** [comment]
It's just a mapping of the sensors configurated "type" to the according DPT subclass. 

As the sensor types become a lot now, maybe we should find a way to configure them by DPT number (additional to the current strings)? @Julius2342

**Comment by farmio** [comment]
If we find a good assignment method we could even ditch a lot of the RemoteValue classes if we just initialize RemoteValueSensor with the right type. Would be less code to maintain.

File: xknx/remote_value/remote_value_sensor.py [hunk 7]
```diff
@@ -24,43 +50,139 @@ class RemoteValueSensor(RemoteValue):
...
         'illuminance': DPTLux,
+        'kelvin_per_percent': DPTKelvinPerPercent,
+        'length': DPTLength,
+        'length_mm': DPTLengthMm,
+        'light_quantity': DPTLightQuantity,
+        'long_delta_timesec': DPTLongDeltaTimeSec,
+        'luminance': DPTLuminance,
         'luminous_flux': DPTLuminousFlux,
+        'luminous_intensity': DPTLuminousIntensity,
+        'lux': DPTLux,
```
Comment (farmio) [inline xknx/remote_value/remote_value_sensor.py]: this is already defined as 'illuminance'

File: xknx/remote_value/remote_value_sensor.py [hunk 8]
```diff
@@ -24,43 +50,139 @@ class RemoteValueSensor(RemoteValue):
...
+        'magnetic_flux_density': DPTMagneticFluxDensity,
+        'magnetic_moment': DPTMagneticMoment,
+        'magnetic_polarization': DPTMagneticPolarization,
+        'magnetization': DPTMagnetization,
+        'magnetomotive_force': DPTMagnetomotiveForce,
+        'mass': DPTMass,
+        'mass_flux': DPTMassFlux,
+        'mol': DPTMol,
+        'momentum': DPTMomentum,
+        'parts_per_million': DPTPartsPerMillion,
```
Comment (farmio) [inline xknx/remote_value/remote_value_sensor.py]: this is already defined as 'ppm'

**Pull request merged**